EsLintの警告を解消する。

**Is your feature request related to a problem? Please describe.**  

以下、警告表示される。
```
  187:17  warning  Can use style directives instead  svelte/prefer-style-directive
  231:18  warning  Can use style directives instead  svelte/prefer-style-directive
  252:8   warning  `{@html}` can lead to XSS attack  svelte/no-at-html-tags
```

**Describe the solution you'd like**  
早めに解消したい。

増えた。。。

```
% npm run lint  
> fact-checklist@1.0.0 lint
> prettier --check . && eslint .

Checking formatting...
All matched files use Prettier code style!

/Users/bonji/workspace/shuji-bonji/fact-checklist/src/routes/+layout.svelte
  128:29  warning  Unexpected class using the ternary operator  svelte/prefer-class-directive
  131:29  warning  Unexpected class using the ternary operator  svelte/prefer-class-directive
  134:29  warning  Unexpected class using the ternary operator  svelte/prefer-class-directive
  137:29  warning  Unexpected class using the ternary operator  svelte/prefer-class-directive
  153:36  warning  Unexpected class using the ternary operator  svelte/prefer-class-directive
  156:36  warning  Unexpected class using the ternary operator  svelte/prefer-class-directive
  159:36  warning  Unexpected class using the ternary operator  svelte/prefer-class-directive
  162:36  warning  Unexpected class using the ternary operator  svelte/prefer-class-directive

/Users/bonji/workspace/shuji-bonji/fact-checklist/src/routes/about/+page.svelte
  325:39  warning  Can use style directives instead  svelte/prefer-style-directive

/Users/bonji/workspace/shuji-bonji/fact-checklist/src/routes/checklist/[id]/+page.svelte
  187:17  warning  Can use style directives instead  svelte/prefer-style-directive
  231:18  warning  Can use style directives instead  svelte/prefer-style-directive
  252:8   warning  `{@html}` can lead to XSS attack  svelte/no-at-html-tags

/Users/bonji/workspace/shuji-bonji/fact-checklist/src/routes/help/+page.svelte
  243:24  warning  Unexpected class using the ternary operator  svelte/prefer-class-directive
  269:9   warning  `{@html}` can lead to XSS attack             svelte/no-at-html-tags
  278:13  warning  `{@html}` can lead to XSS attack             svelte/no-at-html-tags

✖ 15 problems (0 errors, 15 warnings)
  0 errors and 12 warnings potentially fixable with the `--fix` option.
```

--- a/src/routes/+layout.svelte
+++ b/src/routes/+layout.svelte
@@ -125,16 +125,14 @@
 
 			<!-- デスクトップメニュー -->
 			<div class="nav-menu desktop-menu">
-				<button class="nav-link {isHomePage ? 'active' : ''}" onclick={goToHome}>
-					🏠 ホーム
-				</button>
-				<button class="nav-link {isAboutPage ? 'active' : ''}" onclick={goToAbout}>
+				<button class="nav-link" class:active={isHomePage} onclick={goToHome}> 🏠 ホーム </button>
+				<button class="nav-link" class:active={isAboutPage} onclick={goToAbout}>
 					📖 アプリについて
 				</button>
-				<button class="nav-link {isHelpPage ? 'active' : ''}" onclick={goToHelp}>
+				<button class="nav-link" class:active={isHelpPage} onclick={goToHelp}>
 					❓ ヘルプ・使い方
 				</button>
-				<button class="nav-link {isPrivacyPage ? 'active' : ''}" onclick={goToPrivacy}>
+				<button class="nav-link" class:active={isPrivacyPage} onclick={goToPrivacy}>
 					🔐 プライバシー
 				</button>
 			</div>
@@ -150,16 +148,16 @@
 		<!-- モバイルメニュー -->
 		{#if isMenuOpen}
 			<div class="mobile-menu">
-				<button class="mobile-nav-link {isHomePage ? 'active' : ''}" onclick={goToHome}>
+				<button class="mobile-nav-link" class:active={isHomePage} onclick={goToHome}>
 					🏠 ホーム
 				</button>
-				<button class="mobile-nav-link {isAboutPage ? 'active' : ''}" onclick={goToAbout}>
+				<button class="mobile-nav-link" class:active={isAboutPage} onclick={goToAbout}>
 					📖 アプリについて
 				</button>
-				<button class="mobile-nav-link {isHelpPage ? 'active' : ''}" onclick={goToHelp}>
+				<button class="mobile-nav-link" class:active={isHelpPage} onclick={goToHelp}>
 					❓ ヘルプ・使い方
 				</button>
-				<button class="mobile-nav-link {isPrivacyPage ? 'active' : ''}" onclick={goToPrivacy}>
+				<button class="mobile-nav-link" class:active={isPrivacyPage} onclick={goToPrivacy}>
 					🔐 プライバシー
 				</button>
 			</div>

--- a/src/routes/about/+page.svelte
+++ b/src/routes/about/+page.svelte
@@ -322,7 +322,7 @@
 		</div>
 		<div class="categories-grid">
 			{#each categories as category (category.id)}
-				<div class="category-card" style="--category-color: {category.color}">
+				<div class="category-card" style:--category-color={category.color}>
 					<div class="category-icon">{category.emoji}</div>
 					<div class="category-content">
 						<h3 class="category-title">{category.name}</h3>

--- a/src/routes/checklist/[id]/+page.svelte
+++ b/src/routes/checklist/[id]/+page.svelte
@@ -74,6 +74,15 @@
 	function getItemDelay(index: number): string {
 		return `${index * 0.1}s`;
 	}
+
+	function sanitizeHtml(content: string): string {
+		return content
+			.replace(/</g, '&lt;')
+			.replace(/>/g, '&gt;')
+			.replace(/"/g, '&quot;')
+			.replace(/'/g, '&#x27;')
+			.replace(/\n/g, '<br>');
+	}
 </script>
 
 <svelte:head>
@@ -184,7 +193,7 @@
 							<div class="confidence-meter">
 								<div
 									class="confidence-bar {getConfidenceClass(checklist.confidenceLevel)}"
-									style="width: {checklist.confidenceLevel}%"
+									style:width="{checklist.confidenceLevel}%"
 								></div>
 							</div>
 							<div class="confidence-text">
@@ -228,7 +237,7 @@
 								{#each categoryItems as item, index (item.id)}
 									<div
 										class="item-row {item.checked ? 'checked' : 'unchecked'}"
-										style="animation-delay: {getItemDelay(index)}"
+										style:animation-delay={getItemDelay(index)}
 									>
 										<div class="item-status">
 											{item.checked ? '✅' : '❌'}
@@ -249,7 +258,7 @@
 					<div class="notes-display card">
 						<h2>📝 評価メモ</h2>
 						<div class="notes-content">
-							{@html checklist.notes.replace(/\n/g, '<br>')}
+							{@html sanitizeHtml(checklist.notes)}
 						</div>
 					</div>
 				{/if}

--- a/src/routes/help/+page.svelte
+++ b/src/routes/help/+page.svelte
@@ -215,6 +215,15 @@
 	function setActiveSection(sectionId: string) {
 		activeSection = sectionId;
 	}
+
+	function sanitizeHtml(content: string): string {
+		return content
+			.replace(/</g, '&lt;')
+			.replace(/>/g, '&gt;')
+			.replace(/"/g, '&quot;')
+			.replace(/'/g, '&#x27;')
+			.replace(/\n/g, '<br>');
+	}
 </script>
 
 <svelte:head>
@@ -240,7 +249,8 @@
 				{#each helpSections as section (section.id)}
 					<li class="nav-item">
 						<button
-							class="nav-link {activeSection === section.id ? 'active' : ''}"
+							class="nav-link"
+							class:active={activeSection === section.id}
 							onclick={() => setActiveSection(section.id)}
 						>
 							<span class="nav-emoji">{section.emoji}</span>
@@ -266,7 +276,7 @@
 
 						<div class="section-content">
 							<div class="main-content-text">
-								{@html section.content.replace(/\n/g, '<br>')}
+								{@html sanitizeHtml(section.content)}
 							</div>
 
 							{#if section.subSections}
@@ -275,7 +285,7 @@
 										<div class="sub-section">
 											<h3 class="sub-section-title">{subSection.title}</h3>
 											<div class="sub-section-content">
-												{@html subSection.content.replace(/\n/g, '<br>')}
+												{@html sanitizeHtml(subSection.content)}
 											</div>
 										</div>
 									{/each}

